commands/deploy.go

```diff
@@ -276,7 +299,7 @@ func deployImage(
 		image, registryAuth, language,
 		deployFlags.replace, envvars, network,
 		deployFlags.constraints, deployFlags.update, deployFlags.secrets,
-		labelMap, functionResourceRequest1)
+		labelMap, nil, functionResourceRequest1)
```

Comment:
What about passing an empty version of this instead or a named nil variable? (for readability)